### ISSUE TITLE
About Ungoogled chromium v.92.0.4515.131 x86-x64 

### ISSUE THREAD

**Issue by Rusenche** [macchrome/winchrome#51]
![Ungoogled chromium v 92 0 4515 131 x64](https://user-images.githubusercontent.com/18578128/128086831-cabdda85-e5bf-4c84-8928-77b65681ddae.png)
![Ungoogled chromium v 92 0 4515 131 x86](https://user-images.githubusercontent.com/18578128/128086835-05508b53-b4ca-464e-b46c-96cb9ba5a5cf.png)

What does it mean "Version 92.0.4515.131 (Official build, **ununseptium**) (64-bit/32-bit)"?!

**Ungoogled chromium** or **ununseptium**?

**Comment by macchrome** [comment]
ununseptium is Ungoogled chromium; but safe-browser has been disabled by code policies rather than by removing the safe-browsing code; thus resulting in faster release builds.  Check out chrome://safe-browsing/

Basically, since I cross-compile ununseptium for Windows the 2 Windows specific (which are easy to maintain)  patches that need to be applied are:

ungoogled-chromium/windows/windows-disable-encryption.patch
ungoogled-chromium/windows/windows-disable-machine-id.patch

 ununseptium is just a fork of ungoogled chromium with safe-browser disabled by hard-coded code policies.

**Comment by Rusenche** [comment]
Thanks for the reply.

Until yesterday, I had not encountered "**ununseptium**". I had absolutely no idea what it was and whether it was just a compilation error.

About chrome://safe-browsing/:
```
Disabled: safebrowsing.enabled
Disabled: safebrowsing.extended_reporting_opt_in_allowed
Disabled: safebrowsing.scout_reporting_enabled
Disabled: safebrowsing.enhanced
```

In practice, I could not translate for myself what "**ununseptium**" is.

I guess others like me have noticed the change of inscription or are asking the same question.

Thank you for the work you do for everyone and for the correct attitude. Many of us are not familiar with how you make new versions of Ungoogled chromium and have to ask when there is a change that we are not even aware of if it is just a mistake.

In other words - should we expect that in the future "**Ungoogled chromium**" will be dropped (as it was written in **chrome://settings/help** - so it was until the previous version) the inscription /name/ and will be replaced by "**ununseptium**"?

I'm asking because I'm using "**Chrlauncher v.2.5.7**" and I've set it to search for "**Ungoogled chromium**" and the names are really confusing.... I don't know if you understood me clearly or if I explained it correctly?!
